src/simulator/network.c

```diff
@@ -127,7 +129,7 @@ network_step(Network* network, NetworkInputs* inputs, u32 time) {
     for (i = 0; i < network->n_layers; ++i) {
         layer = network->layers[i];
         if (layer->it_ran == FALSE)
-            layer_step(layer, time);
+            layer_step(layer, time, arena, n_cpus);
     }
```

Comment:
Probably don't need this function to change if we pass the pool directly to the layers